jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/MBeanHelper.groovy

```diff
@@ -106,7 +125,9 @@ class MBeanHelper {
         return [ofInterest, attributes].combinations().collect { pair ->
             def (bean, attribute) = pair
             try {
-                new Tuple3(bean, attribute, bean.getProperty(attribute))
+                def extractedAttribute = bean.getProperty(attribute)
```

Comment:
I would expect this to throw the AttributeNoteFoundException for custom ones. As is this would require all passed maps to be value-ovewriting processors instead of allowing a general computed value capability.

Reply:
Right, I guest when I first conceptualized this I only thought about it through the lens of transforming existing attributes. Do you think it would be good to allow the instrument helpers to target attributes that don't actually exist on the mbeans?

Reply:
@rmfitzpatrick wanted to get your thoughts